From 823af8e13d94e6466535fd81b233e8e4b4da2035 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20Br=C3=BCns?= <stefan.bruens@rwth-aachen.de>
Date: Mon, 1 Jun 2020 14:05:46 +0200
Subject: [PATCH] Add missing QPainterPath include required with Qt 5.15

The header is no longer pulled in by QPainter, omitting it causes
failing builds due to incomplete type QPainterPath.
---
 src/qt/dc.cpp       | 1 +
 src/qt/graphics.cpp | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/qt/dc.cpp b/src/qt/dc.cpp
index 5f1324abe9..e304b54c37 100644
--- a/src/qt/dc.cpp
+++ b/src/qt/dc.cpp
@@ -15,6 +15,7 @@
 #include <QtGui/QBitmap>
 #include <QtGui/QPen>
 #include <QtGui/QPainter>
+#include <QtGui/QPainterPath>
 
 #ifndef WX_PRECOMP
     #include "wx/icon.h"
diff --git a/src/qt/graphics.cpp b/src/qt/graphics.cpp
index 9e0614807d..1c4d654acc 100644
--- a/src/qt/graphics.cpp
+++ b/src/qt/graphics.cpp
@@ -19,6 +19,7 @@
 #include <QApplication>
 #include <QDesktopWidget>
 #include <QPainter>
+#include <QPainterPath>
 #include <QPicture>
 
 #ifndef WX_PRECOMP
-- 
2.26.2

